pyquidax/wrappers/accounts.py

```diff
@@ -5,12 +5,33 @@
 
 
 class Account(BaseAPIWrapper):
+    """
+    A class that represent Account and has a BaseAPIWrapper as the object
+    """
+
     def __init__(self, secret_key: Optional[str] = None):
+        """
+        Initializes a new Account instance
+
+        Args:
+            secret_key (Optional[str]): The secret key used for authentication
```

Comment:
There's no need to add type annotations in the docstrings since the method is already annotated. So `secret_key: The secret...` is just fine